markdown/assignments/pool_a/skills/index.md

```diff
@@ -7,12 +7,19 @@ weight: 3
 hidden: true
 ---
 
+## Ziel
+
+In dieser Aufgabe sollen Sie verschiedene Fähigkeiten implementieren, die dann vom Spieler verwendet werden können.
 
 ## Fähigkeiten
 
-Entwickeln Sie verschiedene Fähigkeiten die der Spieler im laufe des Spiels freischalten und benutzen kann.
-Überlegen Sie sich ein geeignetes System um die Fähigkeiten auszurüsten, anzuzeigen, freizuschalten und zu benutzen.
-Implementieren Sie mindestens drei unterschiedliche Fähigkeiten, wobei mindestens eine Kampffähigkeit und eine Non-Kampffähigkeit dabei sein muss.
-Eine Fähigkeit soll einen Cooldown bekommen.
-Eine Fähigkeit soll eine Form von Kosten haben (Geld,Leben,Mana o.ä).
-Denken Sie daran: Sie werden vermutlich noch viele unterschiedliche Fähigkeiten einbauen wollen, daher sollten Sie sich ein gutes Konzept für die Implementierung überlegen.
+Entwickeln Sie verschiedene Fähigkeiten, die der Spieler im Laufe des Spiels freischalten und benutzen kann.
+
+Überlegen Sie sich ein geeignetes System, um die Fähigkeiten auszurüsten, anzuzeigen, freizuschalten und zu benutzen.
+
+Implementieren Sie mindestens drei unterschiedliche Fähigkeiten, wobei mindestens eine Kampffähigkeit und eine Non-Kampffähigkeit dabei sein muss.
+Außerdem: 
+-    Eine der Fähigkeiten soll einen Cooldown bekommen. Es muss also eine gewisse Zeit vergehen, bevor die Fähigkeit erneut eingesetzt werden kann.
+-    Eine der Fähigkeiten soll eine Form von Ressourcenkosten haben. Sie verbraucht also Lebenspunkte, Ausdauerpunkte, `[Manapunkte]({{<ref "https://de.wikipedia.org/wiki/Mana_(Spiele)" >}})`{=markdown} o.ä.
```

Comment:
```suggestion
-    Eine der Fähigkeiten soll eine Form von Ressourcenkosten haben. Sie verbraucht also Lebenspunkte, Ausdauerpunkte, [Manapunkte](https://de.wikipedia.org/wiki/Mana_(Spiele)) o.ä.
```

Reply:
ok, jetzt bin ich verwirrt. Dachte du brauchst dieses `ref` Zeugs für Hugo oderso.

Reply:
jain :) ... Für externe Verweise bitte einfach normale Markdown-Links nehmen. Die internen Verweise gehen mit dieser seltsamen Hugo-Ref-Notation, die dann noch mit dem Schutz vor Pandoc versehen werden muss.

Reply:
sorry. das hätte ich dir mal erzählen können ... aber grad bei diesem hugo-mist gibt es so viel, was man irgendwie beachten muss. langsam fällt das für mich auch in die kategorie "libgdx" ;)

Reply:
ich jetzt wissen, danke ^^